packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { FunctionComponent } from 'react';
 import { storiesOf } from '@storybook/react';
-import { ColorPicker } from '@grafana/ui';
-import { withInfo } from '@storybook/addon-info';
+import { ColorPickerPopover } from './ColorPickerPopover';
```

Comment:
Maybe rename the file to ColorPickerPopover.story.tsx?

Reply:
Will do!